src/cal.py

```diff
@@ -21,4 +21,32 @@
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
+
+calendar.setfirstweekday(calendar.SUNDAY)
+now = datetime.now()
+
+while True:
+  m = input('Enter month: ') or now.month
```

Comment:
FYI, the spec asked for the month and year to come from the command line arguments instead of a user prompt, but you are on the right track.  Love the try/except.